File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -77,375 +78,332 @@ private void InstallDependecies(string shellFilePath)
+   }
+   </div>
 
-	//Initialize HTML to PDF converter 
-	HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+   {% endhighlight %}
 
-	BlinkConverterSettings settings = new BlinkConverterSettings();
+   Include the following namespaces and code samples in the controller for converting HTML to PDF.
 
-	//Set command line arguments to run without sandbox.
-	settings.CommandLineArguments.Add("--no-sandbox");
+   {% highlight c# tabtitle="C#" %}
 
-	settings.CommandLineArguments.Add("--disable-setuid-sandbox");
+   using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using System.IO
 
-	//Assign WebKit settings to the HTML converter 
-	htmlConverter.ConverterSettings = settings;
+   {% endhighlight %}
 
-	//Convert HTML string to PDF
-	PdfDocument document = htmlConverter.Convert("http://www.syncfusion.com");
+   {% highlight c# tabtitle="C#" %}
 
-	//Save the document into stream
-	MemoryStream stream = new MemoryStream();
-
-	document.Save(stream);
-
-	stream.Position = 0;
-
-	//Close the document
-	document.Close(true);
-
-	//Defining the ContentType for pdf file
-	string contentType = "application/pdf";
-
-	//Define the file name
-	string fileName = "URL_to_PDF.pdf";
-
-	//Creates a FileContentResult object by using the file contents, content type, and file name
-	return File(stream, contentType, fileName);
-
-}
+        public ActionResult ExportToPDF()
+        {
+            Environment.SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Development");
+            //Initialize HTML to PDF converter 
+            HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+            BlinkConverterSettings settings = new BlinkConverterSettings();
+            //Set command line arguments to run without sandbox.
+            settings.CommandLineArguments.Add("--no-sandbox");
+            settings.CommandLineArguments.Add("--disable-setuid-sandbox");
+            //Assign WebKit settings to the HTML converter 
+            htmlConverter.ConverterSettings = settings;
+            //Convert HTML string to PDF
+            PdfDocument document = htmlConverter.Convert("http://www.syncfusion.com");
+            //Save the document into stream
+            MemoryStream stream = new MemoryStream();
+            document.Save(stream);
+            stream.Position = 0;
+            //Close the document
+            document.Close(true);
+            //Defining the ContentType for pdf file
+            string contentType = "application/pdf";
+            //Define the file name
+            string fileName = "URL_to_PDF.pdf";
+            //Creates a FileContentResult object by using the file contents, content type, and file name
+            return File(stream, contentType, fileName);
+        }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-### Refer to the following steps to publish as Azure App Linux:
+## Refer to the following steps to publish as Azure App Linux
```

Comment:
Change it as "Steps to publish as Azure App Linux"

Reply:
Changed the content